DragAndDrop/View.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  View.swift
+//  DragAndDrop
+//
+//  Created by Stanley Chiang on 3/6/16.
+//  Copyright © 2016 SantaClaraiOSConnect. All rights reserved.
+//
+
+import UIKit
+
+protocol ViewDelegate {
+    func reset()
+}
+
+class View: UIView, draggableViewDelegate {
+
+    var draggableView:DraggableView!
```

Comment:
More ImplicitlyUnwrappedOptionals :worried: 
Just initialize your views here. 

Also let all the things that you can. 
`let draggableView = initDraggableView()`